Source/RxSwift/mapMany.swift

```diff
@@ -16,7 +16,7 @@ extension ObservableType where E: Collection {
      - returns: An observable collection whose elements are the result of invoking the transform function on each element of source.
      */
 
-    public func mapMany<T>(_ transform: @escaping (E.Element) throws -> T) -> Observable<[T]> {
+    public func mapMany<T>(_ transform: @escaping (E.Element) throws -> T) rethrows -> Observable<[T]> {
```

Comment:
There's no need for a regrow retrow IMO.
Look at the original signature from RxSwift map:

<img width="511" alt="e7bbb63d-ca2e-4f8e-b984-df05005af0b1" src="https://user-images.githubusercontent.com/605076/39676611-5aa688e2-5176-11e8-9e9b-261a8d983256.png">

Reply:
Don't mind reverting the commit, but a func can be a rethrowing func to show that it throws an error only if it’s function parameter throws an error. 🤔

Reply:
Uhm if that would be the case than the map throws without rethrow doesn’t make sense ? I think in general we’d want it to behave much like a regular map so I think It’s best to keep the signature similar